src/Data/Extensible.hs

```diff
@@ -10,22 +10,23 @@
 --
 -- * Basic
 --
---     * 'Data.Extensible.Class': basic membership operations.
```

Comment:
Explanation of these changes:
These links are currently not working as you might have intended. Try clicking any of them here https://hackage.haskell.org/package/extensible-0.9.2/docs/Data-Extensible.html and you'll see it takes you back to the same page.

The proper way to link to modules is to surround their names in double quotes as explained in haddock documentation https://haskell-haddock.readthedocs.io/latest/markup.html#linking-to-modules